applications/StructuralMechanicsApplication/custom_elements/spring_damper_element_3D2N.cpp

```diff
@@ -113,16 +109,18 @@ void SpringDamperElement3D2N::GetDofList( DofsVectorType& rElementalDofList, Pro
 {
     //NEEDED TO DEFINE THE DOFS OF THE ELEMENT
 
-    rElementalDofList.resize( 0 );
-
-    for ( std::size_t i = 0; i < GetGeometry().size(); ++i)
-    {
-        rElementalDofList.push_back( GetGeometry()[i].pGetDof( DISPLACEMENT_X ) );
-        rElementalDofList.push_back( GetGeometry()[i].pGetDof( DISPLACEMENT_Y ) );
-        rElementalDofList.push_back( GetGeometry()[i].pGetDof( DISPLACEMENT_Z ) );
-        rElementalDofList.push_back( GetGeometry()[i].pGetDof( ROTATION_X ) );
-        rElementalDofList.push_back( GetGeometry()[i].pGetDof( ROTATION_Y ) );
-        rElementalDofList.push_back( GetGeometry()[i].pGetDof( ROTATION_Z ) );
+    // Resizing as needed
+    if ( rElementalDofList.size() != msElementSize )
+        rElementalDofList.resize( msElementSize );
+
+    for ( SizeType i = 0; i < msNumberOfNodes; ++i ) {
+        const SizeType index = i * msNumberOfNodes * msDimension;
+        rElementalDofList[index] = GetGeometry()[i].pGetDof( DISPLACEMENT_X);
+        rElementalDofList[index + 1] = GetGeometry()[i].pGetDof( DISPLACEMENT_Y);
```

Comment:
this one is not using flags?

Reply:
This is not my element. And I thought this was already merged via other PR